build.cake

```diff
@@ -97,16 +86,26 @@ Task("Run-Tests")
 
 Task ("NuGet")
 	.IsDependentOn ("Run-Tests")
- 	.WithCriteria(isRunningOnAppVeyor)
 	.Does (() =>
 {
-  AppVeyor.UpdateBuildVersion(string.Format("{0}-{1}-build{2}", version.ToString(), AppVeyor.Environment.Repository.Branch, AppVeyor.Environment.Build.Number));
+  //AppVeyor.UpdateBuildVersion(string.Format("{0}-{1}-build{2}", version.ToString(), AppVeyor.Environment.Repository.Branch, AppVeyor.Environment.Build.Number));
+
+  var nugetVersion = "2.0.0";//AppVeyor.Environment.Repository.Branch == "master" ? version.ToString() : version.Change(prerelease: "pre" + AppVeyor.Environment.Build.Number).ToString();
```

Comment:
Is this temporary? Since nugets are generated every build, I think it's good to keep the pre-release thingy. Could be done on a separate task that only runs on appveyor. And could still use the `version` variable.